test/src/sdk-harness/test_artifacts/tx_contract/src/main.sw

```diff
@@ -2,22 +2,22 @@ contract;
 
 use std::{
     address::Address,
```

Comment:
`Address` no longer needed here with the prelude :)

Reply:
I get a compile error if I don't include this line, even after updating to the latest forc version

Reply:
Not sure if the forc version with the prelude has been released yet... might be only on master?

Reply:
Wait, this test _is_ being added on `master`... why do we do need a release `forc`?